src/Entities/Payments/SofortPayment.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Entities\Payments;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @since 6.0
+ *
+ * @ORM\Table(name="payment_sofort")
```

Comment:
If the persistence is truly private to each BC, then this does not matter. You cannot have a naming conflict with something in another "namespace" (in this case persistence mechanism). I suppose having the prefix anyway might make getting there easier though, so will add it.